hack/swift/Makefile

```diff
@@ -190,6 +190,27 @@ swift-up: rg-up swift-net-up ## Bring up a SWIFT AzCNI cluster
 		--yes
 	@$(MAKE) set-kubeconf
 
+windows-cniv1-up: rg-up overlay-net-up ## Bring up a Windows CNIv1 cluster
```

Comment:
this is not swift.. should we rename directory?

Reply:
That would require a change in all the places using this file ? Is that fine ?

Reply:
Updated the folder to `azcli` instead of `swift`.